### PR TITLE
Refactor variance thresholding

### PR DESCRIPTION
**Pull request by jackyho112**
Related to https://github.com/UBC-MDS/software-review/issues/33#issuecomment-602066695 and https://github.com/UBC-MDS/software-review/issues/33#issuecomment-602118897

### CODE REVIEW

File: R/variance_thresholding.R [hunk 1]
```diff
@@ -22,15 +22,14 @@ variance_thresholding <- function(data, threshold = 0) {
...
 
   selected_feature_indexes <- c()
-  i <- 1
-  for (name in names(data)) {
+  for (i in 1:length(names(data))) {
+    # Get variance of a column
     var_i <- stats::var(data[,i])
 
     if (var_i > threshold) {
+      # Add the column to the selected pile if variance's above the threshold
```
Comment (techrah) [inline R/variance_thresholding.R]: > ... if variance's above ...

Does this mean "if variance is above"? If so, can you change it to avoid confusion?

**Review by techrah — CHANGES_REQUESTED** [review]

**Comment by jackyho112** [comment]
Tests are failing. I'm checking.

File: R/variance_thresholding.R (hunk 1 above)
Comment (techrah) [reply R/variance_thresholding.R]: Ok, I see the fix. Thanks.

**Review by techrah — APPROVED** [review]
Looks good!

**Comment by jackyho112** [comment]
I will add a test

**Pull request merged**